internal/cli/deployments/setup.go

```diff
@@ -254,9 +255,22 @@ func (opts *SetupOpts) configureContainer(ctx context.Context) error {
 	}
 
 	// This can be a high number because the container will become unhealthy before the 10 minutes is reached
-	const healthyDeploymentTimeout = 3 * time.Minute
+	const healthyDeploymentTimeout = 10 * time.Minute
 
-	return opts.WaitForHealthyDeployment(ctx, healthyDeploymentTimeout)
+	err = opts.WaitForHealthyDeployment(ctx, healthyDeploymentTimeout)
+	if err != nil {
```

Comment:
this actually works but initially I was expecting to tail logs 🤔

Reply:
When something fails, don't you want to see the entire log instead of just the runner logs? Or am I misunderstanding your comment?